src/content/docs/en/guides/backend/supabase.mdx

```diff
@@ -143,6 +143,7 @@ Create these endpoints in the `src/pages/api/auth` directory of your project. Yo
 `register.ts` creates a new user in Supabase. It accepts a `POST` request with the an email and password. It then uses the Supabase SDK to create a new user.
 
 ```ts title="src/pages/api/auth/register.ts"
+export const prerender = false;
```

Comment:
```suggestion
// With `output: 'hybrid'` configured:
// export const prerender = false;
```
Maybe something like this for these lines? This line is not necessarily correct for every project.